app/services/foreman_rh_cloud/cloud_request_forwarder.rb

```diff
@@ -0,0 +1,75 @@
+require 'rest-client'
+
+module ForemanRhCloud
+  class CloudRequestForwarder
+    include ::ForemanRhCloud::CloudAuth
+
+    def forward_request(original_request, controller_name, branch_id)
+      forward_params = prepare_forward_params(original_request, branch_id)
+      Rails.logger.debug{"Request parameters for telemetry request: #{forward_params}"}
```

Comment:
We probably need to use one of the foreman loggers here, with a proper label, like here: https://github.com/theforeman/foreman_rh_cloud/blob/master/lib/foreman_inventory_upload/async/shell_process.rb#L39